server/index.js

```diff
@@ -20,6 +22,30 @@ const jsonMiddleWare = express.json();
 app.use(staticMiddleware);
 app.use(jsonMiddleWare);
 
+app.get('/api/parksCache/:parkCode', (req, res, next) => {
+  const parkCode = req.params.parkCode;
+  if (!parkCode) {
+    throw new ClientError(400, 'Park Code must be provided');
+  }
```

Comment:
`req.params.parkCode` is guaranteed to be truthy, otherwise the pattern `/api/parksCache/:parkCode` would not match the request.